js/chrome-tabs.js

```diff
@@ -202,10 +205,14 @@
       this.cleanUpPreviouslyDraggedTabs()
       this.layoutTabs()
       this.setupDraggabilly()
+      return tabEl
     }
 
     setTabCloseEventListener(tabEl) {
-      tabEl.querySelector('.chrome-tab-close').addEventListener('click', _ => this.removeTab(tabEl))
+      const tab = tabEl.querySelector('.chrome-tab-close')
+      if(tab !== null){
```

Comment:
Please conform to the existing code style, e.g. `if(tab !== null){` → `if (tab !== null) {`.